### DISCUSSION TITLE
Netradiant Custom - Can't seem to have multiple folder depth

### DISCUSSION THREAD

**Discussion by Ark771** [func-godot/func_godot_plugin#61] [Q&A] [answered]
When I use func-godot with Netradiant Custom, I can get it to pull my texture folder, but it only goes one single folder deep.
Is this something currently not supported, or is it a setup issue on my end? Netradiant is a weird beast to setup.

My current folder structure.
![image](https://github.com/user-attachments/assets/6d829ba1-8fc4-4376-b59e-7ed34a0a54ad)

How Netradiant shows it.
![image](https://github.com/user-attachments/assets/aba55dc0-1679-46ce-bb16-f834be7e362f)

An example of Quake 3's texture folder, which would be how multiple folder depths should work.
![image](https://github.com/user-attachments/assets/abdcdd89-dc1e-4e85-91fc-a4111ba645e6)

**Comment by RhapsodyInGeek** [comment]
@sinewavey maybe you have some insight into this?

**Comment by sinewavey** [comment, marked as answer]
NetRadiant Custom supports this by sibling folders, matching on prefix:

![image](https://github.com/user-attachments/assets/41fc1002-8189-4159-a092-7b3e4d00a23f)

**Comment by RhapsodyInGeek** [comment]
That's such a funny and weird way of doing it, but thanks for the lesson!
